Part_2/Search/344.cpp

```diff
@@ -0,0 +1,112 @@
+#include <iostream>
+#include <set>
+
+class Matrix
+{
+    double** matrix;
+    int length;
+    int width;
+
+public:
+    Matrix(int a_length, int a_width)
+    {
+        length = a_length;
+        width = a_width;
+
+        matrix = new double* [length];
+        
+        for (int i = 0; i < length; i++)
+        {
```

Comment:
Please update the code following the convention.

Reply:
Which convention is it?